agentconfig/agentconfig.go

```diff
@@ -690,6 +703,11 @@ func RestartFile() string {
 	return restartFileLinux
 }
 
+// OldRestartFile is the location of the restart required file.
+func OldRestartFile() string {
+	return restartFileLinux
```

Comment:
oldRestartFileLinux?

Reply:
That would have been bad, good catch. I will be running extensive tests before this gets pushed FYI.